config/config.exs

```diff
@@ -0,0 +1,4 @@
+use Mix.Config
+
+config :ex_unit,
+  assert_receive_timeout: 1000
```

Comment:
I know `assert_receive_timeout` option behaviour. :)
Sadly, I can't get failures on my machine, could you try 500?
The following failure is not connected to the receive timeout issue:
```
Rollbax.ClientTest: failure on setup_all callback, tests invalidated
```
It means we should explicitly stop `Rollbax.Client` in that test's `setup_all`.